CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## Version 2.2
+
```

Comment:
It looks like this section still needs to be written?

Reply:
Ah yeah. I started thinking I should write it, then changed my mind because I didn't know what date to put on it, and decided I'd write it after the PR was merged. But then I left it like this. I could just write the section and put tomorrow's date on it.